bcipy/tasks/tests/test_main_frame.py

```diff
@@ -0,0 +1,100 @@
+import unittest
```

Comment:
Let's remove the commented out statements.

Reply:
Got it!